app/src/main/java/org/mozilla/fenix/search/toolbar/ToolbarView.kt

```diff
@@ -58,9 +59,21 @@ class ToolbarView(
     override val containerView: View?
         get() = container
 
-    val view: BrowserToolbar = LayoutInflater.from(container.context)
-        .inflate(R.layout.component_search, container, true)
-        .findViewById(R.id.toolbar)
+    val view: BrowserToolbar = if (container.context.settings().shouldUseBottomToolbar) {
+        LayoutInflater.from(container.context)
+            .inflate(R.layout.component_bottom_browser_toolbar, container, true)
+            .findViewById(R.id.toolbar_bottom)
+    } else {
+        if (container.context.settings().shouldUseFixedToolbar) {
+            LayoutInflater.from(container.context)
+                .inflate(R.layout.component_browser_top_toolbar, container, true)
```

Comment:
I think this needs to += `_fixed`.  If not, please write a comment explaining why.

Reply:
👌 fixed by updating as your other comment mentioned.